deployments/farallon/config/prod.yaml

```diff
@@ -1,4 +1,6 @@
 daskhub:
```

Comment:
The top level items on these yaml files are the charts specified as dependencies in `requirements.yaml` - so `daskhub` and `jupyterhub-ssh`. So `jupyterhub-ssh` should be top level, not under `daskhub`.

We have `jupyterhub` and `dask-gateway` under `daskhub` because those two charts are dependencies of `daskhub` - https://github.com/dask/helm-chart/blob/master/daskhub/Chart.yaml#L7